packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -145,4 +145,14 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * Should be used if it's possible to change some properties on items so a strict compare will fail
    */
   itemsAreEqual?: (item1?: any, item2?: any) => boolean;
+
+  /**
+   * A localized string to read out when a selection is deleted
+   */
+  selectionDeletedText?: string;
```

Comment:
I think this should be a prop on the UnifiedPicker instead. It's not being used here, it's being passed through, so should just put it on what's using it

Reply:
Removed.